docs/usage.md

```diff
@@ -6,58 +6,80 @@
 ```
 
-### Full samplesheet
+| Column   | Description                                                                                                                                                |
+| -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `sample` | Custom sample name. This will be used to name all output files from the pipeline. Spaces in sample names are automatically converted to underscores (`_`). |
+| `fasta`  | Full path to a gzipped or uncompressed FASTA file. Accepted file suffixes are: `.fasta`, `.fna`, or `.fa`, or any of these with `.gz`, e.g. `.fa.gz`.      |
+
+An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
-The pipeline will auto-detect whether a sample is single- or paired-end using the information provided in the samplesheet. The samplesheet can have as many columns as you desire, however, there is a strict requirement for the first 3 columns to match those defined in the table below.
+## Databases and reference files
 
-A final samplesheet file consisting of both single- and paired-end data may look something like the one below. This is for 6 samples, where `TREATMENT_REP3` has been sequenced twice.
+nf-core/funcscan utilises various tools that use databases and reference files to generate results. While nf-core/funcscan offers in some cases functionality to autodownload databases for you, these databases can be very large, and it is more efficient to store these files in a central place and can be reused across pipeline runs.
 
-```console
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
-CONTROL_REP2,AEG588A2_S2_L002_R1_001.fastq.gz,AEG588A2_S2_L002_R2_001.fastq.gz
-CONTROL_REP3,AEG588A3_S3_L002_R1_001.fastq.gz,AEG588A3_S3_L002_R2_001.fastq.gz
-TREATMENT_REP1,AEG588A4_S4_L003_R1_001.fastq.gz,
-TREATMENT_REP2,AEG588A5_S5_L003_R1_001.fastq.gz,
-TREATMENT_REP3,AEG588A6_S6_L003_R1_001.fastq.gz,
-TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,
+Here we will describe where you can obtain databases and reference files for tools included in the pipeline.
+
+### hmmsearch
+
+nf-core/funcscan allows screening of sequences for various natural product types via hidden markov models with HMMSearch.
+
+This requires supplying a list of HMM model files ending in `.hmm`, that have models for the particular molecule(s) you are interested in.
+
+You can download such files from places such as [https://pfam.xfam.org/](https://pfam.xfam.org/) for antimicrobial peptides (AMP), or the antiSMASH github repository for [biosynthetic gene cluster](https://github.com/antismash/antismash/tree/master/antismash/detection/hmm_detection/data) related HMM models
+
+You should place all of these in a directory and can be supplied e.g. for AMP models
```

Comment:
```suggestion
You should place all of these in a directory and supply them e.g. to AMP models
```
or fix the connection in some other way (between "place .. these" and "[they] can be supplied")